os/binfmt/libelf/libelf_addrenv.c

```diff
@@ -77,6 +80,36 @@
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
+#ifdef CONFIG_OPTIMIZE_APP_RELOAD_TIME
+static void allocateregions(FAR struct elf_loadinfo_s *loadinfo)
```

Comment:
How about adding `_` between two words? `allocate_regions`?
For better readability, could you add "what region" in function name?